app/controllers/invitation_controller.rb

```diff
@@ -65,8 +79,20 @@ def auto_complete_for_user_name
     @users = User.where("LOWER(name) LIKE ?", "%#{search}%") unless search.blank?
   end
 
+  def accepted_invitation(user_email)
```

Comment:
Hello Prof, we have fixed all the naming conventions that you have suggested us.
accept_invitation => email_an_invitation
accepted_invitation => email_an_acceptance
student_email => student_id